examples/SavedModel-fashion/ormbfile.yaml

```diff
@@ -0,0 +1,2 @@
+author: "Ce Gao <gaoce@caicloud.io>"
+format: SavedModel
```

Comment:
Should we fill in more information here?

Reply:
Now I think it is enough. We can add more info soon. Because model config is not determined now.

Reply:
LGTM